lib/systems/include/PedalsSystem.h

```diff
@@ -109,38 +107,18 @@ class PedalsSystem
     /// @param params
     /// @param max_percent_diff
     /// @return
-    bool evaluate_pedal_implausibilities_(int pedalData1_analog,
-                                          int pedalData2_analog,
+    bool evaluate_pedal_implausibilities_(int pedal_1_raw,
```

Comment:
Does raw mean analog? Does raw mean non-digitally-filtered? Please avoid the use of "raw" as much as possible. **be descriptive**.